src/screens/CreatePost/index.js

```diff
@@ -384,6 +356,10 @@
     setListTopic(newArr);
     setHashtags(newArr);
     setListTopicChat(newChat);
+
+    if (!newArr.map((i) => i.topic).includes(params?.topic)) {
+      setSelectedTopic(null);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setSelectedTopic` state is being set to null if the current topic is not included in the new array of topics. This is a good practice to ensure that the selected topic always exists within the available topics.
```diff
+ if (!newArr.map((i) => i.topic).includes(params?.topic)) {
+   setSelectedTopic(null);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->